main.tf

```diff
@@ -171,10 +171,61 @@ resource "azurerm_key_vault" "wwt" {
 data "azurerm_client_config" "current" {
 }
 
-# Give the VM access to the KeyVault via Managed Identity
+resource "azurerm_app_service_plan" "wwt" {
+  name                = "wwt-appserviceplan"
```

Comment:
I think it's probably a good idea to apply the same prefix variable here for the service plan name

Reply:
Fixed